src/palace/manager/api/axis.py

```diff
@@ -1423,14 +1410,6 @@ def _raise_exception_on_error(
 
 
 class XMLResponseParser(ResponseParser, Axis360Parser[T], ABC):
```

Comment:
The changes here make the XMLResponseParser subclasses return the parsed data directly, rather then a LoanInfo or HoldInfo object. Previously they returned a partial object, and the caller had to fill in the missing details, now the caller creates the whole LoanInfo or HoldInfo object.